### PR TITLE
Fix:SPCTR-1848-Container-Initial layout selection UI in editor and customizer block editor

### PR DESCRIPTION
**Pull request by ravinishadcoder**
### Description

Fix: Container-Initial layout selection UI in editor and customizer block editor
[Task link](https://brainstormforce.atlassian.net/browse/SPCTR-1848?atlOrigin=eyJpIjoiZTFlNjgwMDE1ZjhiNDlhMWFkZTU5YzM1OGVlZjY4ZWYiLCJwIjoiaiJ9)
### Screenshots

[Screencast](https://d.pr/v/t9bTMT)
### Types of changes

Bug fix (non-breaking change which fixes an issue) 

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: src/blocks/container/edit.js [hunk 1]
```diff
@@ -184,18 +186,29 @@ const UAGBContainer = ( props ) => {
...
+					icon={ UAGB_Block_Icons.container }
+					label={ __( 'Container', 'ultimate-addons-for-gutenberg' ) }
+					instructions={ ReactHtmlParser(
+						sprintf(
+							// translators: %s: closing </br> tag.
+							__(
+								'Customizable containers with endless creation possibilities.%sSelect a container layout to start with.',
+								'ultimate-addons-for-gutenberg'
+							),
+							`</br>` // eslint-disable-line no-undef
```
Comment (bsf-zanev) [inline src/blocks/container/edit.js:209]: Why are we using no-undef for this break tag...?

File: src/blocks/container/editor.lazy.scss [hunk 2]
```diff
@@ -53,10 +51,23 @@
...
+			margin-bottom: 4px;
+			color: #1e1e1e;
+		}
+	}
+
+	.components-placeholder__label {
+		path,
+		rect,
+		circle {
+			stroke: #0063a1;
```
Comment (bsf-zanev) [inline src/blocks/container/editor.lazy.scss:66]: Let's use `$spectra-color-secondary` here.

File: src/blocks/container/editor.lazy.scss [hunk 3]
```diff
@@ -53,10 +51,23 @@
...
 		&.is-large .components-placeholder__label,
 		.components-placeholder__label {
-			font-size: 18px;
+			font-size: 24px;
 			font-weight: 400;
-			margin-bottom: 0;
-			color: #556068;
+			line-height: 29px;
+			margin-bottom: 4px;
+			color: #1e1e1e;
```
Comment (bsf-zanev) [inline src/blocks/container/editor.lazy.scss:58]: Confirm with designers if we need to add this color to our palette, or if we can use one of our existing colors from the [Figma Spectra Color Palette](https://www.figma.com/file/Adn6EAVPWePS5BuB86p5ng/Spectra-Style-Guide?node-id=109-155&t=ipm90EaTDNrQX2oz-0).

File: src/blocks/container/editor.lazy.scss [hunk 4]
```diff
@@ -53,10 +51,23 @@
...
+
+	.components-placeholder__label {
+		path,
+		rect,
+		circle {
+			stroke: #0063a1;
+		}
+
+		svg {
+			margin-right: 11.5px;
```
Comment (bsf-zanev) [inline src/blocks/container/editor.lazy.scss:70]: We can make this 12px.

File: src/blocks/container/editor.lazy.scss [hunk 5]
```diff
@@ -66,14 +77,21 @@
 
 	.components-placeholder__fieldset {
 		align-items: center;
+		text-align: center;
+		
+		.components-placeholder__instructions {
+			font-size: 13px;
+			line-height: 20px;
+			color: #50575e;
```
Comment (bsf-zanev) [inline src/blocks/container/editor.lazy.scss:85]: Let's use `$spectra-color-body` here.

File: src/blocks/container/editor.lazy.scss [hunk 6]
```diff
@@ -146,17 +169,28 @@
...
 			}
 		}
 	}
 }
 
+@media only screen and ( max-width: 412px ) {
+	.uagb-container-variation-picker {
+
+		.components-placeholder__fieldset {
+			max-width: 100% !important;
```
Comment (bsf-zanev) [inline src/blocks/container/editor.lazy.scss:185]: Why is important used here?

**Review by bsf-zanev — CHANGES_REQUESTED** [review]

File: src/blocks/container/editor.lazy.scss (hunk 6 above)
Comment (ravinishadcoder) [reply src/blocks/container/editor.lazy.scss:185]: `BlockVariationPicker` which we are using from  `@wordpress/block-editor`, has set default `max-width` for `components-placeholder__fieldset`, to overwrite that default CSS, I have used` ! important` there.

File: src/blocks/container/edit.js (hunk 1 above)
Comment (ravinishadcoder) [reply src/blocks/container/edit.js]: I have removed that no-undef

File: src/blocks/container/editor.lazy.scss (hunk 2 above)
Comment (ravinishadcoder) [reply src/blocks/container/editor.lazy.scss]: Done the suggested changes.

File: src/blocks/container/editor.lazy.scss (hunk 4 above)
Comment (ravinishadcoder) [reply src/blocks/container/editor.lazy.scss]: Done the suggested changes.

File: src/blocks/container/editor.lazy.scss (hunk 5 above)
Comment (ravinishadcoder) [reply src/blocks/container/editor.lazy.scss]: Done the suggested changes.

File: src/blocks/container/editor.lazy.scss (hunk 3 above)
Comment (ravinishadcoder) [reply src/blocks/container/editor.lazy.scss]: I have confirmed with Akbar Hartomo and used the color from the [Figma Spectra Color Palette](https://www.figma.com/file/Adn6EAVPWePS5BuB86p5ng/Spectra-Style-Guide?node-id=109-155&t=ipm90EaTDNrQX2oz-0).

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: src/blocks/container/edit.js [hunk 7]
```diff
@@ -185,18 +187,29 @@ const UAGBContainer = ( props ) => {
...
+					icon={ UAGB_Block_Icons.container }
+					label={ __( 'Container', 'ultimate-addons-for-gutenberg' ) }
+					instructions={ ReactHtmlParser(
+						sprintf(
+							// translators: %s: closing </br> tag.
+							__(
+								'Customizable containers with endless creation possibilities.%sSelect a container layout to start with.',
+								'ultimate-addons-for-gutenberg'
+							),
+							`</br>` // eslint-disable-line
```
Comment (bsf-zanev) [inline src/blocks/container/edit.js:210]: Is this eslint comment required?

**Review by bsf-zanev — APPROVED** [review]
Good to go, just remove that Eslint-disable-lint and test once.

File: src/blocks/container/edit.js (hunk 7 above)
Comment (ravinishadcoder) [reply src/blocks/container/edit.js]: I have removed this line

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**